index.html

```diff
@@ -0,0 +1,150 @@
+            <h1 class="homeText">SOS! Save Our Planet</h1>        
+            <h1 class="home-header">save our planet global summit 2022</h1>
+            <p class="someText">
+                Global temperatures and sea levels are rising, 
+                and possibly contributing to larger more devastating storms.
+                This can all be contributed to climate change. 
+                Climate change is defined as gradual changes in all the interconnected weather elements on our planet over approximately 30 years. 
+                The data shows the Earth is warming and it's up to us to make the changes necessary for a healthier planet.
+            </p>
+            <p class="h-text">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from 
+                over 80 countries is taking place in August, in Paris.
+            </p>
+
+            <h2 class="date">2022.08.15(MON) ~ 16(TUE)</h2>
+            <p class="venue">@ Conference Hall, Paris</p>
+        </section>
+
+        <section id="program" class="prog">
+            <h2 class="prog-header">Main Program</h2>
+            <hr class="prog-line">
+            <ul class="row-container">
+                <li class="lecture">
+                    <img src="./assets/lecture.png" alt="Lecture Icon on IconScout">
+                    <h3>Lecture</h3>
+                    <p>Listen to popular speakers from various countries about the next steps and necessary actions the world has to take to fight climate change.</p>
+                </li>
+                <li class="exhibition">
+                    <img src="./assets/whiteboard.png" alt="Explaining concepts on a board">
+                    <h3>Exhibition</h3>
+                    <p>Appreciate various efforts made so far towards the fight against climate change. Showcase marvelous works in that light.</p>
+                </li>
+                <li class="forum">
+                    <img src="./assets/speech-bubble.png" alt="Share your thoughts with others">
+                    <h3>Forum</h3>
+                    <p>Have the time to share your thoughts and opinions with experts for each topic. Ask and answer popular questions.</p>
+                </li>
+                <li class="workshop">
+                    <img src="./assets/screwdriver_318-881772.png" alt="Work with others to build stuff practically ">
+                    <h3>Workshop</h3>
+                    <p>Examine new ways of conquering climate change by checking for safer or more efficient energy production techniques.</p>
+                </li>
+                <li class="connect">
+                    <img src="./assets/networking.png" alt="Connect with others">
+                    <h3>Networking</h3>
+                    <p>Get amazing opportunities to network with SOP affiliates around the world, also after the summit. Work better together.</p>
+                </li>
+            </ul>
```

Comment:
- [Optional] Remember these are the articles section. I think it's preferable to make use of an article element (`<article></article>`) for each article instead of listing them with `<ul></ul>`. This will be more semantically correct.